checker/mta_sts.go

```diff
@@ -121,7 +128,8 @@ func checkMTASTSPolicyFile(domain string, hostnameResults map[string]HostnameRes
 		}
 	}
 	defer resp.Body.Close()
-	body, err := ioutil.ReadAll(resp.Body)
+	// Read up to 10,000 bytes of response body.
+	body, err := ioutil.ReadAll(io.LimitReader(resp.Body, 10000))
```

Comment:
Looks like the recommended limits are 64 kb / 1min, though it's not necessary to adhere to this.
https://tools.ietf.org/html/rfc8461#page-11